md6/src/consts.rs

```diff
@@ -0,0 +1,21 @@
+/// MD6 constants related to standard mode of operation
+
+pub(crate) type Md6Word = u64;
```

Comment:
You may just use `pub` for items in this module since you use `pub(crate) mod consts;` in lib.rs.